### PR TITLE
Intial layout

### PR DESCRIPTION
**Pull request by tom-parker**

### CODE REVIEW

**Comment by keirog** [comment]
:+1:

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
```
Comment (matthew-andrews) [inline gulpfile.js]: you don't need gulpfile.js any more :)

File: package.json [hunk 2]
```diff
@@ -13,10 +13,12 @@
   "devDependencies": {
     "chai": "^1.9.2",
     "gulp": "^3.8.8",
```
Comment (matthew-andrews) [inline package.json]: you don't need gulp any more :)

File: package.json [hunk 3]
```diff
@@ -13,10 +13,12 @@
...
-    "mocha": "^1.21.4",
+    "gulp-notify": "^2.2.0",
```
Comment (matthew-andrews) [inline package.json]: or this

File: package.json [hunk 4]
```diff
@@ -13,10 +13,12 @@
...
+    "mocha": "^2.0.1",
     "next-build-tools": "^v1.18.1",
+    "next-gulp-tasks": "Financial-Times/next-gulp-tasks.git#v1",
```
Comment (matthew-andrews) [inline package.json]: don't need this either

**Pull request merged**